backend/transcribee_backend/base/models.py

```diff
@@ -16,7 +21,44 @@ class Document(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     changed_at = models.DateTimeField(auto_now=True)
 
+    def __str__(self):
+        return self.name
+
 
 class DocumentUpdate(models.Model):
     document = models.ForeignKey(Document, on_delete=models.CASCADE)
     update_content = models.BinaryField()
+
+
+def generate_worker_token():
+    return get_random_string(
+        length=128, allowed_chars=string.ascii_lowercase + string.digits
+    )
+
+
+class Worker(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    token = models.CharField(max_length=128, default=generate_worker_token)
+    name = models.CharField(max_length=255, null=False, blank=False)
+
+    last_seen = models.DateTimeField()
+
+
+class Task(models.Model):
```

Comment:
Is there some way we can split the API definition for the tasks out of the backend, so they can be reused by the worker implementation?

Reply:
I don't think it makes sense to split this out, as these are django model definitions. But it might make sense to add the pydantic version of this to the `proto`s